packages/react-inbox/src/components/Inbox/index.tsx

```diff
@@ -180,14 +175,25 @@ const Inbox: React.FunctionComponent<InboxProps> = (props) => {
   const handleIconOnClick = useCallback(
     (event: React.MouseEvent) => {
       event.preventDefault();
+      let myCurrentTab = currentTab;
       if (!isOpen) {
         setView("messages");
-        setCurrentTab(tabs?.[0] ?? DEFAULT_TABS[1]);
+        myCurrentTab = tabs?.[0] ?? DEFAULT_TABS[1];
+        setCurrentTab(myCurrentTab);
       }
 
+      if (!lastMessagesFetched) {
```

Comment:
will probably do something with this to "refetch" if a certain time has elapsed but can deal with that in later optimiation